docs/pods.md

```diff
@@ -55,3 +55,11 @@ _Why not just run multiple programs in a single (Docker) container?_
 _Why not support affinity-based co-scheduling of containers?_
 
 That approach would provide co-location, but would not provide most of the benefits of pods, such as resource sharing, IPC, guaranteed fate sharing, and simplified management.
+
+## Durability of pods (or lack thereof)
+
+Pods should never be treated as durable pets. They won't survive scheduling failures, node failures, or other evictions, such as due to lack of resources or for node maintenance. In general, users shouldn't create pods directly. They should almost always use controllers (e.g., replication controller), even for singletons.
+
+Pod is exposed as a primitive to facilitate writing schedulers, controllers, etc., to facilitate bootstrapping, for separation of concerns -- Kubelet vs. cluster-level components, to facilitate decoupling of replication controllers and services, and so that replication controllers and other similar controllers don't need to proxy pod-level operations.
+
+The right solution for pets is something like nominal services [#260](https://github.com/GoogleCloudPlatform/kubernetes/issues/260). We expect controllers to replace pods in advance of their termination and certainly in advance of deletion (e.g., for planned evictions, image prefetching, unidling, or live pod migration [#3949](https://github.com/GoogleCloudPlatform/kubernetes/issues/3949)).
```

Comment:
I have a small problem saying "the right solution" and then linking to something that isn't implemented.